docs/eventing/sources/_index.md

```diff
@@ -38,7 +38,7 @@ kn source list
 | [AWS SQS](https://github.com/knative-sandbox/eventing-awssqs/blob/master/pkg/apis/sources/v1alpha1/aws_sqs_types.go)  | v1alpha1 | Knative | Brings [AWS Simple Queue Service](https://aws.amazon.com/sqs/) messages into Knative. The AwsSqsSource fires a new event each time an event is published on an [AWS SQS topic](https://aws.amazon.com/sqs/).  |
 | [Apache Camel](https://github.com/knative-sandbox/eventing-camel/blob/master/pkg/apis/sources/v1alpha1/camelsource_types.go) | v1alpha1   | Knative    | Enables use of [Apache Camel](https://github.com/apache/camel) components for pushing events into Knative. A CamelSource is an event source that can represent any existing [Apache Camel component](https://github.com/apache/camel/tree/master/components), that provides a consumer side, and enables publishing events to an addressable endpoint. Each Camel endpoint has the form of a URI where the scheme is the ID of the component to use. CamelSource requires [Camel-K](https://github.com/apache/camel-k#installation) to be installed into the current namespace. See the [CamelSource](https://github.com/knative-sandbox/eventing-camel/tree/master/samples) example. |
 | [Apache CouchDB](https://github.com/knative-sandbox/eventing-couchdb/blob/master/source)                                                   | v1alpha1 | Knative    | Brings [Apache CouchDB](https://couchdb.apache.org/) messages into Knative.  |
-| [Apache Kafka](https://github.com/knative/eventing-contrib/blob/master/kafka/source/pkg/apis/sources/v1alpha1/kafka_types.go)       | v1beta1  | Knative    | Brings [Apache Kafka](https://kafka.apache.org/) messages into Knative. The KafkaSource reads events from an Apache Kafka Cluster, and passes these events to a sink so that they can be consumed. See the [Kafka Source](https://github.com/knative/eventing-contrib/tree/master/kafka/source) example for more details.  |
+| [Apache Kafka](https://github.com/knative-sandbox/eventing-kafka/blob/master/pkg/apis/sources/v1alpha1/kafka_types.go)       | v1beta1  | Knative    | Brings [Apache Kafka](https://kafka.apache.org/) messages into Knative. The KafkaSource reads events from an Apache Kafka Cluster, and passes these events to a sink so that they can be consumed. See the [Kafka Source](https://github.com/knative-sandbox/eventing-kafka/blob/master/pkg/source) example for more details.  |
```

Comment:
Should this be pointing to:
.../pkg/apis/sources/v1beta1/kafka_types.go

instead of v1alpha1?

Reply:
yeah,that looks more appropriate when pointing to b1beta1